src/main/java/org/jvnet/hudson/test/junit/jupiter/JenkinsRule.java

```diff
@@ -0,0 +1,19 @@
+package org.jvnet.hudson.test.junit.jupiter;
+
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * JUnit 5 meta annotation providing {@link org.jvnet.hudson.test.JenkinsRule} integration.
```

Comment:
I think it would be helpful if you can add an example snippet of a test case in the JavaDoc here (or in `JenkinsRuleExtension`). So one sees how to correctly create such a test method.

Reply:
Done, examples and some additional documentation added to `@JenkinsRule` – typically this should be used mainly.